From 7a103a971c95e9f700926361dcdbacbb4c81470b Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 25 Jun 2018 13:50:12 +0800
Subject: [PATCH 4125/5242] MLK-18428-02: doc: binding: Add binding doc for
 imx8mm tmu

commit  a9113b9a9a2484398e246cfce0a4e9798614fe88 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts binding doc for i.MX8MM TMU.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/thermal/imx8mm-thermal.txt |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/thermal/imx8mm-thermal.txt

diff --git a/Documentation/devicetree/bindings/thermal/imx8mm-thermal.txt b/Documentation/devicetree/bindings/thermal/imx8mm-thermal.txt
new file mode 100644
index 0000000..4a9d0c5
--- /dev/null
+++ b/Documentation/devicetree/bindings/thermal/imx8mm-thermal.txt
@@ -0,0 +1,17 @@
+* Temperature Monitor (TMU) on Freescale i.MX8MM SoCs
+
+Required properties:
+- compatible : "fsl,imx8mm-tmu"
+- reg: Address and length of the register
+- clocks : thermal sensor's clock source.
+- #thermal-sensor-cells : Should be 0. See ./thermal.txt for a description.
+Example:
+
+tmu: tmu@0x30260000 {
+	compatible = "fsl,imx8mm-tmu";
+	reg = <0x0 0x30260000 0x0 0x10000>;
+	clocks = <&clk IMX8MM_CLK_TMU_ROOT>;
+	interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>;
+	#thermal-sensor-cells = <0>;
+};
+
-- 
1.7.9.5

